pkg/kncloudevents/message_sender_test.go

```diff
@@ -149,9 +154,9 @@ func TestHTTPMessageSenderSendWithRetries(t *testing.T) {
 	}}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			var n atomic.Int32
+			var n int32
```

Comment:
Why change these? Should be equivalent :thinking:

Reply:
I prefer to use the built in whenever is possible

Reply:
IMO the actual types are nicer as they prevent you from holding things wrong. But if you prefer otherwise, that's fine too. I usually opt for as much help via the typesystem as possible and IIRC these do not carry a runtime cost.